app/src/main/AndroidManifest.xml

```diff
@@ -9,17 +9,17 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name="org.lnmiit.activities.MainActivity"
+            android:name="lnmiit.android.app.MainActivity"
```

Comment:
have you forgot to create folder for activity? please create one and put all the activities into it.